core/src/action.rs

```diff
@@ -65,15 +68,18 @@ impl Hash for ActionWrapper {
 pub enum Action {
     /// entry to Commit
     /// MUST already have passed all callback checks
-    Commit(Entry),
-    /// hash to Get
-    Get(String),
-
+    CommitEntry(Entry),
+    /// hash of Entry to get
+    GetEntry(HashString),
```

Comment:
are we doing `Entry` and dropping the `Get` in our naming conventions?

Reply:
Why dropping the `Get`? Isn't that almost a stronger characteristic of this action than `Entry`?

Reply:
For this part I was sticking to the API in HDK rust:  `hdk::get_entry()`